### DISCUSSION TITLE
Is Lively Windows 11 compatible?

### DISCUSSION THREAD

**Discussion by ghost** [rocksdanister/lively#779] [Q&A] [answered]
Is this software compatible with Windows 11?

**Comment by rocksdanister** [comment, marked as answer]
Yes 🥳

https://blogs.windows.com/windowsexperience/2021/10/04/11-things-to-know-about-the-new-microsoft-store-on-windows-11/

>Want to customize your PC? Check out popular utilities like PowerToys, EarTrumpet or Lively Wallpaper.

**Reply by ghost** [reply]
Good 👍
